zerver/openapi/zulip.yaml

```diff
@@ -1459,7 +1459,13 @@ paths:
                                 value:
                                   description: |
                                     The new value of the changed property.
+
+                                    **Changes**: Starting with Zulip 10.0 (feature level 314), this
+                                    field can be an object for `can_remove_subscribers_group` property
+                                    when the setting is set to a combination of users and groups.
                                   oneOf:
+                                    - type: object
```

Comment:
Not sure how to handle this. Only defining `type` without properties and `additionalProperties` does not work, and adding all the info results in this -
![image](https://github.com/user-attachments/assets/29484e58-57ee-4a2c-b0e9-fbf99ffa9d0e)

Reply:
Hmm, yeah, that's messy.

Reply:
I think we can go with something like this:

> If an object, it will be a [group-setting value]() with these fields:

Reply:
Updated.